installation/aws/VmClarity.cfn

```diff
@@ -322,43 +390,45 @@ Resources:
                       -p 0.0.0.0:1000:80/tcp \
                       ${FreshclamMirrorContainerImage}
 
                     [Install]
                     WantedBy=multi-user.target
                   - FreshclamMirrorContainerImage: !If [FreshclamMirrorContainerImageOverridden, !Ref FreshclamMirrorContainerImageOverride, "ghcr.io/openclarity/freshclam-mirror:v0.1.0"]
               mode: "000644"
-          commands:
-            01subsitute_rest_address:
-              command: /etc/vmclarity/render_config.sh
-            02reload_systemctl:
-              command: systemctl daemon-reload
-
-            03enable_exploit_db_server:
-              command: systemctl enable exploit-db-server.service
-            04start_restart_exploit_db_server:
-              command: systemctl restart exploit-db-server.service
-
-            05enable_trivy_server:
-              command: systemctl enable trivy_server.service
-            06start_restart_trivy_server:
-              command: systemctl restart trivy_server.service
-
-            07enable_grype_server:
-              command: systemctl enable grype_server.service
-            08start_restart_grype_server:
-              command: systemctl restart grype_server.service
+            "/lib/systemd/system/postgres.service":
+              content:
+                Fn::Sub:
+                  - |
+                    [Unit]
+                    Description=Postgresql Database Server
+                    After=docker.service
+                    Requires=docker.service
 
-            09enable_vmclarity_freshclam_mirror:
-              command: systemctl enable vmclarity_freshclam_mirror.service
-            10start_restart_vmclarity_freshclam_mirror:
-              command: systemctl restart vmclarity_freshclam_mirror.service
+                    [Service]
+                    TimeoutStartSec=0
+                    Restart=always
+                    ExecStartPre=-/usr/bin/docker stop %n
+                    ExecStartPre=-/usr/bin/docker rm %n
+                    ExecStartPre=/usr/bin/docker pull ${PostgresqlContainerImage}
+                    ExecStart=/usr/bin/docker run \
+                      --rm --name %n \
+                      --network vmclarity \
+                      -e POSTGRESQL_USERNAME=vmclarity \
+                      -e POSTGRESQL_PASSWORD=${PostgresDBPassword} \
+                      -e POSTGRESQL_DATABASE=vmclarity \
+                      -p 127.0.0.1:5432:5432/tcp \
+                      ${PostgresqlContainerImage}
 
-            11enable_vmclarity:
-              command: systemctl enable vmclarity.service
-            12start_restart_vmclarity:
-              command: systemctl restart vmclarity.service
+                    [Install]
+                    WantedBy=multi-user.target
+                  - PostgresqlContainerImage: !If [PostgresqlContainerImageOverridden, !Ref PostgresqlContainerImageOverride, "bitnami/postgresql:12.14.0-debian-11-r28"]
```

Comment:
`bitnami/postgresql:12.14.0-debian-11-r28` maybe we want to use the official images that are less vulnerable?